### ISSUE TITLE
Float rounding and casting causes IllegalArgumentException in Bitmap.createBitmap in specific cases

### ISSUE THREAD

**Issue by simonpercic** [Yalantis/uCrop#73]
When using specific cropping settings with specific image sizes on some phones, a crash occurs:

```java
java.lang.IllegalArgumentException: y + height must be <= bitmap.height()
	at android.graphics.Bitmap.createBitmap(Bitmap.java:689)
	at android.graphics.Bitmap.createBitmap(Bitmap.java:653)
	at com.yalantis.ucrop.view.CropImageView$override.cropImage(CropImageView.java:138)
```

##### Steps to reproduce:

1. open the sample app included with the library
- set the aspect ratio to square
- set image resize to max size of 1024x1024
- tap pick & crop and select an image from the gallery (or from the camera, doesn't matter) of a specific size (see below)
- don't change the cropping region, rotation or pan (basically, don't change anything in the cropping activity)
- tap on "done" in the cropping activity
- the activity crashes and the app displays a toast with the exception message

##### What combinations do not work?

- Nexus 5X with image size: 4032x3024 (the largest 4:3 camera photo size)
- Nexus 6P with image size: 4032x3024 OR 2688x1512 (the largest 4:3 and the middle 16:9 camera photo sizes, respectively)
- Nexus 6 with image size: 4032x3024 OR 2688x1512

The issue might be present on other phone and image size combinations as well.

Since this happens on the actual photo sizes the user might have selected in their camera app, this is a probable real-world scenario.

##### Why does this happen?

Basically, float rounding and then casting to int causes the resized image to be 1 pixel too small before cropping.
This happens in the CropImageView.cropImage() method.

Occurs in the 1.3.2 version of the library.

##### Screenshots

Settings: 
![device-2016-04-06-192249](https://cloud.githubusercontent.com/assets/9164783/14327200/8b42cfd4-fc31-11e5-8f94-89e53000f24f.png)

Toast message following the exception:
![device-2016-04-06-192338](https://cloud.githubusercontent.com/assets/9164783/14327216/9a9a8a12-fc31-11e5-94b8-567242cf8fcb.png)

**Comment by shliama** [comment]
Thanks for the report. I'll work on the issue today.

**Comment by shliama** [comment]
Fixed with your PR #74
